pkgs/development/compilers/manticore/default.nix

```diff
@@ -1,15 +1,15 @@
 { stdenv, fetchFromGitHub, coreutils, autoreconfHook, smlnj }:
 
 let
-  rev= "47273c463fc3c5d0a0ae655cf75a4700bdb020b4";
+  rev= "4528ccacdfd53d36f5959c005b27cd7ab6175b83";
```

Comment:
Shall we just put this in with the rest of `fetchFromGitHub` while we’re here? Not sure what it’s doing up here.